### ISSUE TITLE
Like hyperlink , can it detect phone number and turn Bold ?

### ISSUE THREAD

**Issue by santhanakrishnanstark** [dzonatan/angular2-linky#12]

**Comment by santhanakrishnanstark** [comment]
Or any other package like this, suggest me ?

**Comment by dzonatan** [comment]
Have you tried using it? Looking at [Autolinker.js options](https://github.com/gregjacobs/Autolinker.js#options) it looks like it should detect phone numbers out of the box.

**Comment by santhanakrishnanstark** [comment]
@dzonatan  yes, its ok but i need the format like only numbers without space like 9876543210 , but its not affect , how can i apply for this contineous digit numbers ?
